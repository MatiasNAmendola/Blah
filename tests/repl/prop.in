a: []
[].length
a.length
a.push('a')
a
[].push('b', 3, 'c')
a.length
a.push()
a.push('ab', 3, 'ab')
a.pop()
a
b: [1, [], 12]
a.push(b)
a
b: a.push
c: a.pop
a.pop()
c()
b('abc')
a
a.length
