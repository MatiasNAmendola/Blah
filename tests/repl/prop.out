0
0
Nothing
['a']
Nothing
1
Nothing
Nothing
ab
['a', 'ab', 3]
Nothing
['a', 'ab', 3, [1, [], 12]]
[1, [], 12]
3
Nothing
['a', 'ab', 'abc']
3
